recipe/meta.yaml

```diff
@@ -1,60 +1,71 @@
 {% set name = "hip" %}
-{% set version = "2.7.2" %}
+{% set version = "6.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  # Need git for version info generation in cmake
-  git_url: https://github.com/ROCm-Developer-Tools/HIP.git
-  git_tag: roc-{{ version }}
-  #url: https://github.com/ROCm-Developer-Tools/HIP/archive/roc-{{ version }}.tar.gz
-  #sha256: 576ece14983ca257e5b64fedbf08c5a19bfad179d64d971e377eb91dac35a7e6
+  - url: https://github.com/ROCm/HIP/archive/refs/tags/rocm-{{ version }}.tar.gz
+    sha256: b47178db94f2acc106e1a88ceb029844805266ebaba11ef63744e90d224b11be
+    folder: hip
+
+  - url: https://github.com/ROCm/clr/archive/refs/tags/rocm-{{ version }}.tar.gz
+    sha256: cb8ac610c8d4041b74fb3129c084f1e7b817ce1a5a9943feca1fa7531dc7bdcc
+    folder: clr
+    patches:
+      - 0001-Don-t-try-to-install-windows-files-on-linux.patch
+      - 0001-Mark-raw-strings.patch
+      - 0002-Remove-download-of-libamd64.so.5.patch
 
 build:
-  number: 1
+  number: 0
   skip: True  # [not linux]
-  merge_build_host: True
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ cdt('xorg-x11-proto-devel') }}
+    - {{ cdt('libx11-devel') }}
+    - {{ cdt('mesa-libgl-devel') }}
+    - {{ cdt('mesa-libegl-devel') }}
+    - sysroot_linux-64 2.17  # [linux64]
     - cmake
     - make
-    - perl *
+    - cppheaderparser
+    - wget
   host:
-    - rocr-runtime         {{ ".".join(version.split(".")[:2]) }}
-    - rocm-device-libs     {{ ".".join(version.split(".")[:2]) }}
-    - roct-thunk-interface {{ ".".join(version.split(".")[:2]) }}
-    - rocm-comgr           {{ ".".join(version.split(".")[:2]) }}
-    - hcc                  {{ version }}
-    - clangdev 9.0.*
-    - llvmdev 9.0.*
-    - llvm
-    - perl *
+    - clangdev 17.*
+    - hipcc                {{ version }}
+    - libnuma
+    - llvmdev  17.*
+    - ocl-icd
+    - rocm-cmake           {{ version }}
+    - rocm-comgr           {{ version }}
+    - rocr-runtime         {{ version }}
+    - zlib
   run:
-    - {{ pin_compatible("rocr-runtime", max_pin="x.x") }}
-    - {{ pin_compatible("rocm-comgr", max_pin="x.x") }}
-    - {{ pin_compatible("roct-thunk-interface", max_pin="x.x") }}
-    - hcc                  {{ version }}
-    - rocminfo
-    - perl *
+    - rocm-comgr           {{ version }}
+    - rocr-runtime         {{ version }}
 
 test:
+  requires:
+    - hipcc                {{ version }}
   commands:
-    - $PREFIX/bin/hipcc --amdgpu-target=gfx803 --version
-    - $PREFIX/bin/hipify-clang --version
+    - $PREFIX/bin/hipcc --offload-arch=gfx803 --version
     - $PREFIX/bin/hipconfig
 
 about:
-  home: https://github.com/ROCm-Developer-Tools/HIP
+  home: https://github.com/ROCm/HIP
   license: NCSA
   license_family: MIT
-  license_file: LICENSE
-  summary: 'HIP : Convert CUDA to Portable C++ Code'
+  license_file:
+    - clr-LICENSE
```

Comment:
```suggestion
    - clr/LICENCE
```

Reply:
I tried it like that first, but got a complaint from the checker, see [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=901237&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=2562).

Reply:
You had `clr/LICENSE` not `clr/LICENCE`

Reply:
D'oh. Fixed.